production/technical_userguide.md

```diff
@@ -666,3 +666,68 @@ curl https://api.bcda.cms.gov/data/45/99dbc86f-74e5-4553-88a9-af3e718cb72b.ndjso
 The response will be the requested data as [FHIR Coverage resources](https://www.hl7.org/fhir/coverage.html){:target="_blank"} in NDJSON format.
 
 An example of one such resource is available in the [guide to working with BCDA data](/data-guide/#sample-bcda-files).
+
+## Filtering Your Data with `_since`
+### About `_since` 
+`_since`, also known as "the since parameter," grants you the ability to apply a date parameter to your bulk data. Instead of receiving the full record of historical data every time you request data from an endpoint, you will be able to use `_since` to submit a date. BCDA will then produce data from the Patient, Coverage, or Explanation of Benefit endpoints that have been loaded since the entered date.
```

Comment:
The "also known as.." sounds funny to me.  What if we just had "The `_since` parameter grants you..."

Reply:
Followed the suggestion and resolved.